frontend/automation/cypress/e2e/wallets.cy.js

```diff
@@ -35,14 +36,14 @@ describe('create wallet', () => {
         expect(text.split(' ').length).to.equal(24)
       })
     cy.getByTestId('create-wallet-success-cta').click()
-    cy.url().should('contain', `/wallet/${walletName}/keypair`)
-    cy.getByTestId('wallet-name').contains(walletName)
+    cy.getByTestId('header-title').should('have.text', walletName)
   })
 })
 
 describe('wallet', () => {
   let passphrase = ''
   let walletName = ''
+  let pubkey
```

Comment:
Suggestion(non-blocking): empty string for type

Reply:
changed it for consistency.